sandbox/dev_tools/views.py

```diff
@@ -27,7 +27,14 @@ def dev_consumer(request: HttpRequest) -> HttpResponse:
         form = LTIConsumerForm(request.POST)
         if form.is_valid():
             launch_url = request.build_absolute_uri(
-                reverse("lti_provider.views.launch")
+                reverse(
+                    "forum.lti.view",
+                    kwargs={
+                        "uuid": uuid.uuid5(
```

Comment:
uuid5?

Reply:
I use uuid5 + NAMESPACE_DNS to generate a consistent uuid based on a string.
This is to simplify the dev tools interface..